examples/docs_snippets/docs_snippets/guides/components/index/3-pip-tree.txt

```diff
@@ -3,11 +3,14 @@ tree
 .
```

Comment:
I didn't know we but the output of `tree` into code snippets. I may have to update the quickstart